sklearn/decomposition/_kernel_pca.py

```diff
@@ -1,14 +1,15 @@
 """Kernel Principal Components Analysis"""
 
 # Author: Mathieu Blondel <mathieu@mblondel.org>
+#         Sylvain Marie <sylvain.marie@schneider-electric.com>
```

Comment:
just so you know, we don't maintain these anymore, and you're now implicitly accepting to be pinged for anything that goes wrong with this file ;)

Reply:
I dont mind being pinged about this file :) I think that I owe my employer some visibility as I am grateful to have the opportunity to contribute to such a project as part of my work, that's why I wished to add my email address here, mentioning schneider. But really, whatever you prefer or whatever suits best sklearn's policy.